libass/ass_utils.c

```diff
@@ -256,9 +256,9 @@ uint32_t parse_color_tag(char *str)
     return ass_bswap32((uint32_t) color);
 }
 
-uint32_t parse_color_header(char *str)
+int32_t parse_int_header(char *str)
```

Comment:
uint32_t here?

Reply:
No, signed `int32_t` is correct; headers can have negative values. E.g. in the commit you’re commenting on, the VSF-style clamping for negative alpha values `FFMAX(FFMIN(front_alpha, 0xFF), 0)`  wouldn't work *(as straight forward)* using only unsigned integers.
For more info see the corresponding commit which was already merged into master: https://github.com/libass/libass/commit/b7b816a8ea09d26574ba4090b6e1362a59065b66